packages/opentelemetry-exporter-stackdriver-trace/README.md

```diff
@@ -0,0 +1,98 @@
+# OpenTelemetry Stackdriver Trace Exporter
```

Comment:
It would be nice if you can update the trace exporter list: https://github.com/open-telemetry/opentelemetry-js#trace-exporters